packages/runner-ct/package.json

```diff
@@ -41,23 +57,9 @@
     "react-devtools-inline": "^4.10.1",
     "react-dom": "16.8.6",
     "react-split-pane": "^0.1.92",
```

Comment:
`react-split-pane` is not used anymore and should be removed from the dependencies.

Reply:
It seems it is: https://github.com/cypress-io/cypress/blob/97d5a6d88e48c48ef9bc7048a49c9651e313a085/packages/runner-ct/src/app/RunnerCt.tsx#L5

Reply:
I got too deep in the unification... Sry